src/reynir/Greynir.grammar

```diff
@@ -457,9 +467,13 @@ Skilyrðissetning →
     | Skilyrðistenging BeygingarliðurMeðUmröðun # 'Ef ekki næðist samkomulag yrði málið svæft'
     | Skilyrðistenging BeygingarliðurÁnF
     | Skilyrðissetning_vh
+    | Skilyrðistenging BeygingarliðurMeðLepp    # Það er rétta aðferðin ef það skapast hættuástand
 
 Skilyrðissetning_vh →
-    TengimerkiVh? SagnliðurVh
+    TengimerkiVh? BeygingarliðurVh
+
+BeygingarliðurVh →
```

Comment:
If you rely on `BeygingarliðurVh` being present when generating SimpleTree or for other purposes, you need to add `$tag(keep) BeygingarliðurVh` - otherwise it will be optimized away, since it simply points to a single nonterminal.

Reply:
Done